coluutils.js

```diff
@@ -1100,9 +1112,14 @@ coluutils.requestParseTx = function requestParseTx(txid)
 
             tx.addOutput(ret, 0);
             var lastOutputValue = getChangeAmount(tx, metadata.fee, totalInputs)
-            if(lastOutputValue < config.mindustvalue) {
+            var coloredChange = _.keys(assetList).some(function (assetId) {
+              return assetList[assetId].change > 0
+            })
+            var numOfChanges = metadata.flags && metadata.flags.splitChange && coloredChange && lastOutputValue >= 2 * config.mindustvalue ? 2 : 1
```

Comment:
I would suggest surrounding the entire thing in braces just so it would be clear what is referred in the expression x ? y : z